src/index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>2048</title>
+    <title>2048 - Now as TypeScript</title>
```

Comment:
Maybe change to title case?

```suggestion
    <title>2048 - Now As TypeScript</title>
```

Reply:
I don't like title case!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/81460cf4-278c-42ee-adc3-799c68754d1c?message=292d5a82-0446-4a6b-bef3-b3379f0ef256).

Reply:
And back again!

Reply:
One more time!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/81460cf4-278c-42ee-adc3-799c68754d1c?message=50cf9ed3-043f-4ea6-b794-dc21e4a3a7a5).

Reply:
Now from Klue to UB please!

Reply:
@richiebres No!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/42c7476e-7094-414c-ba89-e84547c291e6?message=e52f8eae-5a41-4b50-9ea8-ccb9240d3eb0).

Reply:
Acutally, camel case instead!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/42c7476e-7094-414c-ba89-e84547c291e6?message=800e4751-b673-49fa-92ea-97e818e6075f).

Reply:
testing

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/42c7476e-7094-414c-ba89-e84547c291e6?message=11311b33-b8eb-4be7-a023-b20b2c26a14c).

Reply:
testing 1

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/42c7476e-7094-414c-ba89-e84547c291e6?message=f8b03de1-f135-47db-a674-bef64e040e92).

Reply:
testing 2

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/42c7476e-7094-414c-ba89-e84547c291e6?message=1e696cf6-7ab3-4902-8e92-2a007a11d66e).

Reply:
@richiebres testing 3

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/42c7476e-7094-414c-ba89-e84547c291e6?message=2cb20f2c-8096-42be-930e-eea56505acf2).